Ultz.SilkExtensions.ImGui/ImGuiController.cs

```diff
@@ -37,7 +37,7 @@ public class ImGuiController : IDisposable
         /// <summary>
         /// Constructs a new ImGuiController.
         /// </summary>
-        public ImGuiController(GL gl, IView view, IInputContext input)
+        public ImGuiController(GL gl, IView view, IInputContext input, ImGuiFontConfig imGuiFontConfig)
```

Comment:
This is a breaking change. Consider making a second constructor or move the font configuration to a property.